.github/workflows/preview-docs.yml

```diff
@@ -0,0 +1,97 @@
+name: Preview Documentation
+on:
+  pull_request:
+  pull_request_target:
```

Comment:
I think we can get rid of `pull_request_target` altogether.  I assume we want the `types: [labeled]` line under `pull_request`?

Reply:
GitHub action workflow cannot access secret variables by `pull_request` even if it is triggered by a label in my mind. Please let me know if my understanding is incorrect. You probably know more about this than I do.